main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -222,6 +222,14 @@ sealed trait Completion {
         documentation = None,
         kind = CompletionItemKind.Method
       )
+    case Completion.UseEnumTagCompletion(sym, cas) =>
```

Comment:
We write `caze` to avoid the clash with `case`.

Reply:
Ah, I've just seen it as `cas` elsewhere but I'll change it